package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-azurefunctions",
-    "version": "0.24.1",
+    "version": "0.24.2-alpha",
```

Comment:
Please do alpha bump in a separate PR. They're unrelated and the app settings stuff may take longer to review/merge while the alpha bump should be super quick